util/topgen/intermodule.py

```diff
@@ -880,7 +880,9 @@ def im_netname(sig: OrderedDict,
             if obj.get("package") == "tlul_pkg" and obj["struct"] == "tl":
                 return "{package}::{struct}_H2D_DEFAULT".format(
                     package=obj["package"], struct=obj["struct"].upper())
-            return "{package}::{struct}_REQ_DEFAULT".format(
+            # default is used for dangling ports in definitions.
+            # the struct name already has `_req` suffix
+            return "{package}::{struct}_DEFAULT".format(
```

Comment:
@tjaychen I found that dangling port for I-M signal creates wrong default value as it uses definitions. signame in definition has `_req` or `_rsp` suffix already. Not sure how to handle this nicely.

Reply:
ahh.. will need to have a look...might need more condition checks...

Reply:
what should be the correct value in this case? sorry i can't tell which case is wrong.

Reply:
Above change works. But I will review the dangling port handling code and will think what would be a good way to solve this. Or, as it works, we can just leave it for now. :)